debian/control.top.in

```diff
@@ -43,7 +43,8 @@ Build-Depends:
     tcl@TCLTK_VERSION@-dev,
     tclx,
     tk@TCLTK_VERSION@-dev,
-    yapps2
+    yapps2,
+    @QTVCP_DEPENDS@
```

Comment:
Is this really a build-time dependency?  We've been building fine without it so far,  have we been missing something?

Reply:
It is if you wish to use a qtvcp screen immediately upon installing the package without running the aforementioned scrip. They’ve definitely been missing.  I don’t know the rationale except I think it was born during the python2/python3 changeover…

Reply:
It is not required for builds, just a convenience for running QtVCP GUI's from RIP installs.
If this is omitted then we should leave the install script as is.

Reply:
The build environment should be as minimal as possible, with only things actually needed for the build installed.  I'm ok with keeping the install script if that's helpful for your workflow.

Reply:
Sounds reasonable to me.  I removed the changes to debian/control.top.in